OpenRA.Game/FileFormats/VocLoader.cs

```diff
@@ -31,6 +33,7 @@ bool ISoundLoader.TryParseSound(Stream stream, string fileName, out byte[] rawDa
 			{
 				rawData = null;
 				channels = sampleBits = sampleRate = 0;
+				stream.Seek(position, SeekOrigin.Begin);
 				return false;
```

Comment:
You can use a `finally` block to unconditionally reset the stream position when returning.